### PR TITLE
I18n implementation

### PR DESCRIPTION
**Pull request by Zakrok09**

### CODE REVIEW

**Review by Zakrok09 — COMMENTED** [review]
I think as a first implementation it is suffiecient

File: src/lib/translations/translations.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "homepage.first.paragraph3": "‼️ Although we make sure that every link is secure, make sure to check where the link is leading to before clicking!",
+        "homepage.first.box1": "We are students from all over Europe who just started working on a project together. We all want to help people. This is why we developed RoTUer.",
+        "homepage.first.box2": "RoTUer was made to help people like me who just suddenly find themselves in a place with endless possibilities. This is why we wanted to present all this information in a summarized form with links.",
+        "homepage.util1": "About us",
+        "homepage.util2": "About RoTUer",
+        "homepage.util3": "More",
+        "homepage.questions": "Do you want to",
+        "homepage.questions.first": "...book a place to study in peace?",
+        "homepage.questions.second": "...use the most out of your X?",
+        "homepage.questions.third": "...access student portal?",
```
Comment (dnbln) [inline src/lib/translations/translations.js:16]:
```suggestion
```

Those have no reason to be here; as they are related to links they should come from the links json file.

File: src/lib/translations/translations.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        "homepage.fourth.1": "Found a missing link",
+        "homepage.fourth.2": "If the link we are missing",
+        "homepage.fourth.3": "is a legitimate source of information",
+        "homepage.fourth.4": "is the official website of e.g. shops for texbooks and",
+        "homepage.fourth.5": "does not contain any pirated content",
+        "homepage.fourth.6": "Send us an email",
+        "homepage.fourth.7": "Open an issue"
```
Comment (dnbln) [inline src/lib/translations/translations.js:23]: I'm sure there are more more meaningful names we can give to those keys.

File: src/lib/translations/translations.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "homepage.first.paragraph3": "‼️ Hoewel we ervoor zorgen dat elke link veilig is, zorg ervoor dat u controleert waar de link naartoe leidt voordat u klikt!",
+        "homepage.first.box1": "Wij zijn studenten uit heel Europa die net samen aan een project zijn begonnen. We willen allemaal mensen helpen. Daarom hebben we RoTUer ontwikkeld.",
+        "homepage.first.box2": "RoTUer is gemaakt om mensen zoals ik te helpen die zich plotseling op een plek bevinden met eindeloze mogelijkheden. Daarom wilden we al deze informatie in een samengevatte vorm met links presenteren.",
+        "homepage.util1": "Over ons",
+        "homepage.util2": "Over RoTUer",
+        "homepage.util3": "Meer",
+        "homepage.questions": "Wil je",
+        "homepage.questions.first": "...een plek boeken om in alle rust te studeren?",
+        "homepage.questions.second": "...het meeste halen uit je X abonnement?",
+        "homepage.questions.third": "...toegang tot het studentenportaal?",
```
Comment (dnbln) [inline src/lib/translations/translations.js:39]:
```suggestion
```

Same as above.

**Pull request merged**